runtime/doc/lua.txt

```diff
@@ -2473,4 +2473,47 @@ trust({opts})                                             *vim.secure.trust()*
         • true and full path of target file if operation was successful
         • false and error message on failure
 
+
+==============================================================================
+Lua module: semver                                                *lua-semver*
+
+cmp({v1}, {v2}, {opts})                                         *semver.cmp()*
+    Compares two strings ( `v1` and `v2` ) in semver format.
+
+    Parameters: ~
+      • {v1}    (string) Version.
+      • {v2}    (string) Version to be compared with v1.
+      • {opts}  (table|nil) Options to configure parsing behaviour with the
+                optional keys:
+                • strict (boolean): (default = false) see `semver.parse` for
+                  details.
+
+    Return: ~
+        (integer) -1 if `v1 < v2`,0 if `v1 == v2` or, 1 if `v1 > v2`.
+
+parse({version}, {opts})                                      *semver.parse()*
+    Parses a semantically formatted version string into a table.
+
+    Supports leading "v" and leading and trailing whitespace in the version
+    string, that means `"v1.0.1-rc1+build.2"`, `"1.0.1-rc1+build.2"`, `"
+    v1.0.1-rc1+build.2"` and `"v1.0.1-rc1+build.2 "` will all produce the
+    table:
+
+    { major = 1, minor = 0, patch = 1, prerelease = 'rc1 , build = 'build.2' }`
+
+    Parameters: ~
+      • {version}  (string) Version string formatted semantically.
+      • {opts}     (table|nil) Options to configure parsing behaviour which
+                   takes the following optional keys:
+                   • strict (boolean): (default = false) when set to `true` an
```

Comment:
as far as I remember at other places there is written `Defaults to true` and `bool` instead of `boolean` is used

Reply:
Ah yes, you are right. I will amend.

Reply:
I think we should wait for one of the core maintainers then
I just looked at a few helpages and just saw bool being used

Reply:
Use `boolean`.